torchao/utils.py

```diff
@@ -12,7 +12,7 @@
 from functools import reduce
 from importlib.metadata import version
 from math import gcd
-from typing import Any, Callable, Optional, Type
+from typing import Any, Callable, Optional
```

Comment:
please fix rebase to not have these changes, or open a new PR if you don't know how to fix rebase